_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -110,6 +110,20 @@ You will most likely not need to specify any parameters except for `location`. F
    sudo ./bin/opensearch-keystore add s3.client.default.secret_key
    ```
 
+1. (Optional) If you're using a custom S3 endpoint (for example, MinIO), disable AWS EC2 metadata connection:
+
```

Comment:
Line 113: The service is named "Amazon EC2" (never "AWS EC2"). Additionally, if these are the first appearances of these service names, they should be "Amazon Simple Storage Service (Amazon S3) endpoint" and "Amazon Elastic Compute Cloud (Amazon EC2) metadata connection". If these are not the first appearances, "S3 endpoint" and "EC2 metadata connection" are fine.